database/mysql/mysql_test.go

```diff
@@ -175,55 +174,3 @@ func TestLockWorks(t *testing.T) {
 		}
 	})
 }
-
```

Comment:
The removal of these test worry me since they both:
1. list the connection "urls" (actually DSNs) that we support
1. state our understanding of how the go-sql-driver/mysql driver parses DSNs

Can you find a way to incorporate these tests with the new DSN parsing? It may be easiest to move your changes into `urlToMySQLConfig()` instead of removing it.